algosdk/mnemonic.py

```diff
@@ -78,11 +79,13 @@ def to_private_key(mnemonic):
 
 
 def to_public_key(mnemonic):
+    warnings.warn("to_public_key returns the Algorand address and will be deprecated, "
```

Comment:
Could you move this below the docstring, and also add a line to the docstring that says a similar thing?

Reply:
Ah yeah I missed that! Fixed.